### ISSUE TITLE
rows get out of alignment in submission portal

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc-server#1048]
During the NMDC workshop today some users that rows get out of alignment. This makes it difficult for users to tell what row goes with what sample.
<img width="1546" alt="Screenshot 2023-08-24 at 9 04 10 AM" src="https://github.com/microbiomedata/submission-schema/assets/8196591/5d4feef3-567a-4fbf-8c01-d7b93a7e2642">

**Comment by aclum** [comment]
This came up when working with Jeff Blanchard yesterday and I'd like this looked at after GSP. He had data where ever 3 rows was different so the misalignment seems like it could increase data entry errors. 
cc @mslarae13 @pkalita-lbl 
patrick, any idea if this is a straightforward fix or not?

**Comment by pkalita-lbl** [comment]
Solidly not straightforward. 

It's a bug in Handsontable (the spreadsheet interface library that DataHarmonizer uses): https://github.com/handsontable/handsontable/issues/10726. The latest [comment](https://github.com/handsontable/handsontable/issues/10726#issuecomment-1892357910) from one of that project's maintainers says there is no workaround and no specific timeline for fixing the bug.

**Comment by aclum** [comment]
Bummer, I've added the blocked label to this and upvoted the issue you linked.

**Comment by mslarae13** [comment]
For what it's worth. If we could force the columns to always be the width of any text or NOT wrap it would help

**Comment by mslarae13** [comment]
I've figured out why... depending on how and where you copy paste from, some text add 'invisible' characters & put an 'enter' or extra line before or after the sample name or other text..

so it looks like 

"Sample1"

but it's

**Comment by pkalita-lbl** [comment]
Yes, copy-and-pasting is one way to run into the problem. But there are a number ways to get into the same state. A user can manually type newline characters (`option` + `return` on Mac):

https://github.com/microbiomedata/nmdc-server/assets/102547565/6d6ee2cd-021b-4fec-a921-c1771d1d3075

Or a user could type a long string into a cell and then manually change the column width:

https://github.com/microbiomedata/nmdc-server/assets/102547565/894498c7-496b-4320-8f5f-39fdd3d69941

I'd have to read the docs to see what sort of settings Handsontable provides around disabling wrapping or stripping newlines; I don't know off the top of my head. If those settings exist, they may solve _some_ of the problematic cases.
